vexim/adminuserchange.php

```diff
@@ -265,20 +282,20 @@
         </tr>
         <tr>
           <td><?php echo _('Forwarding on'); ?>:</td>
-          <td><input name="on_forward" type="checkbox" <?php
+          <td><input name="on_forward" id="on_forward" type="checkbox" <?php
             if ($row['on_forward'] == "1") {
               print " checked";
-            } ?>>
+            } ?> onchange="fwac()" onclick="fwac()">
           </td>
         </tr>
         <tr>
           <td><?php echo _('Forward mail to'); ?>:</td>
           <td>
-            <input type="text" size="25" name="forward"
+            <input type="text" size="25" name="forward" id="forward"
             value="<?php print $row['forward']; ?>" class="textfield"><br>
             <?php echo _('Must be a full e-mail address'); ?>!<br>
```

Comment:
I think we should change these two labels as well.
E.g. "Enter full e-mail addresses, use commas to separate them".
And then something instead of the "OR", but not sure what right now.